sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -853,7 +853,7 @@ class GpuRLikeMeta(
         case Literal(str: UTF8String, DataTypes.StringType) if str != null =>
           try {
             // verify that we support this regex and can transpile it to cuDF format
-            pattern = Some(new CudfRegexTranspiler(RegexFindMode).transpile(str.toString))
+            pattern = (new CudfRegexTranspiler(RegexFindMode).transpile(str.toString, None))._1
```

Comment:
nit: it isn't obvious when reading this code what `_1` represents - I will follow up with a separate PR to make this clearer